astropy/modeling/core.py

```diff
@@ -801,6 +807,92 @@ def inverse(self):
         raise NotImplementedError("An analytical inverse transform has not "
                                   "been implemented for this model.")
 
+    @property
+    def bounding_box(self):
+        """
+        A `tuple` of length `n_inputs` defining the bounding box limits, or 
+        `None` for no bounding box. 
+
+        The default is `None`, unless a ``bounding_box_default`` method is defined.
+
+        `bounding_box` can be set manually to an array-like  object of shape 
+        ``(2 * model.n_inputs,)`` or ``(model.n_inputs, 2)``.
+
+        .. note:: The limits are ordered according to the `numpy` indexing convention, 
```

Comment:
There should probably be only one place where the axis order for bounding boxes is described.
Either here or in the high-level docs.
Then use a Sphinx label and `:ref:` from one to the other.